Assets/QA/Tests/XRDeviceActions/_SharedScripts/Vector2ControlActionStatus.cs

```diff
@@ -19,6 +20,22 @@ void OnEnable()
         vector2Action.performed += UpdateVector2;
         vector2Action.started += UpdateVector2;
         vector2Action.cancelled += UpdateVector2;
+
+        ReadOnlyArray<InputControl> controls = vector2Action.controls;
+        for (int i = 0; i < controls.Count; i++)
+        {
+            Vector2Control control = controls[i] as Vector2Control;
+            if (control != null)
+            {
+                Vector2 value = control.ReadValue();
+                status1Slider.value = value.x;
```

Comment:
cant tell from the diff, but assume that the values are written back to the vector in the same order from the sliders?